etc/dbus-serialbattery/battery.py

```diff
@@ -250,7 +250,8 @@ def manage_charge_voltage_linear(self) -> None:
         voltageSum = 0
         penaltySum = 0
         tDiff = 0
-
+        # meassurment and variation tolerance in volts
+        measurementToleranceVariation = 0.0215
```

Comment:
I would not use more than 3 decimals. Is there a reason why you chose to use 4?

Reply:
not really, some value between 0.02 and 0.03 should be Ok, that is not in principle. For me was value 0.022 most optimal. Please set any value of you choice.   (I'm afraid to suggest that as a parameter)

Reply:
The optimal value may be related to CELL_VOLTAGE_DIFF_KEEP_MAX_VOLTAGE_UNTIL and also to specific load fluctuations in the entire system.

Reply:
Example with Value 0.03
![Screenshot 2023-06-26 at 22 51 04](https://github.com/Louisvdw/dbus-serialbattery/assets/50322596/36a97b1b-691d-4c96-a1d5-9d4af3649c5a)

With Value 0.0215 it takes some more time to meet conditions (better for battery)
![Screenshot 2023-06-26 at 22 50 33](https://github.com/Louisvdw/dbus-serialbattery/assets/50322596/aeb4b00c-ba6f-4c3a-9755-9f1ded71cc40)